### ISSUE TITLE
GUI problems

### ISSUE THREAD

**Issue by DemianTinkiel** [SecUSo/privacy-friendly-sudoku#69]
grid shows as blank, existing numbers only show when highlighting a row/column.
I tried switching to dark mode and back, as well as disabling highlight (worst, numbers never show).

Also note that the lines between quadrants is the same width as the line between cells
Phone: 
Mi mix 2s
Android 10Q
Mi UI 12.01

App: 3.0.1

![Screenshot_2020-11-21-10-42-58-759_org secuso privacyfriendlysudoku](https://user-images.githubusercontent.com/7367884/99877341-797ab980-2bf5-11eb-9f13-611edd70ee08.jpg)

![Screenshot_2020-11-21-10-43-09-665_org secuso privacyfriendlysudoku](https://user-images.githubusercontent.com/7367884/99877348-7da6d700-2bf5-11eb-83d5-545553ba4f88.jpg)

**Comment by Kamuno** [comment]
Can you try if this also occurs with version v3.0.2?

**Comment by DemianTinkiel** [comment]
@Kamuno is that version available in the playstore? I didn't see it there. 3.0.1 seems to be the latest there
![Screenshot from 2020-12-18 16-26-06](https://user-images.githubusercontent.com/7367884/102637408-de222900-414d-11eb-9f55-388b09a24f14.png)

**Comment by Kamuno** [comment]
Oh, it seems it is only available in the beta track, that is currently disabled... my bad.. I'll push it into the playstore in a bit.

**Comment by lecciovich** [comment]
@Kamuno I have updated to v3.0.2 but problem still be as it is shown by @DemianTinkiel

**Comment by lecciovich** [comment]
Phone settings:
Model: xiaomi poco x3 nfc
Os: Android 10QKQ1.2005.002
Miui: MIUI Global (stabile) 12.0.7

**Comment by lecciovich** [comment]
Solved: My problem was system enabling of dark theme (from Android settings).
Once disabled it works properly.

**Comment by Kamuno** [comment]
Does that mean you are using light theme now? .. Because I would not consider this a fix, really. I think the App should work in dark theme as well.

**Comment by lecciovich** [comment]
You re right. It's more a temporary workaround. Enabling dark theme from Android system settings I meet again the visual problem above.
Btw I found the way to disable dark theme only in the app.

**Comment by annihat** [comment]
v3.0.2 works on Realme 7 but it's unpleasant to use, the highlights are difficult to distinguish which slows down the game 


![Screenshot_2021-03-26-10-02-13-61_8ddc00f2bd4c3524a64d3dec58f9a8e7](https://user-images.githubusercontent.com/25818824/112615703-bb717180-8e1a-11eb-880b-dcf1a571d7d6.jpg)

**Comment by DemianTinkiel** [comment]
> Solved: My problem was system enabling of dark theme (from Android settings).
> Once disabled it works properly.

I agree with @lecciovich. for me what works is
1. In OS (w/ global dark mode enabled), put exception to use normal theme for this app
2. In App enable dark mode

I have MiMix 2S so this might be related specifically to Xiaomi's implementation of dark mode?
